_includes/manuals/1.14/2.1_quickstart_installation.md

```diff
@@ -200,10 +200,15 @@ apt-get update && apt-get -y install foreman-installer
 
 #### Running the installer
 
+If you are planning an installation on an existing system, it's strongly advised to make
+ a backup of your system configuration. This especially applies to apache configuration,
+ since the used puppet module does remove existing configuration files.
```

Comment:
depending on the type of install: at least also postgres/mysql config, puppet config, tftps/xinetd configs, BIND config, DHCPd config.